qiskit/test/mock/utils/backend_converter.py

```diff
@@ -123,6 +124,9 @@ def convert_to_target(conf_dict: dict, props_dict: dict = None, defs_dict: dict
                             target[inst][(qubit,)].calibration = sched
                     else:
                         target[inst][qarg].calibration = sched
+    target.add_instruction(
+        Delay(Parameter("t")), {(bit,): None for bit in range(target.num_qubits)}
```

Comment:
I also noticed that reset is missing while working on https://github.com/Qiskit/red-queen/pull/6 so we should add that here before merging this.

Reply:
Never mind, ignore this. I was running on a fake backend that predated the reset support. Reset is properly supported already